docs/renarde/JavaSupport.md

```diff
@@ -0,0 +1,14 @@
+# Java Support
+
+### REST Endpoints
+
+For Renarde `Controller` classes, Renarde uses a different strategy than JAX-RS to determine which methods count as REST endpoints, and what the path of each endpoint is.
+vscode-quarkus contains functionality so that the existing JAX-RS features (code lens and workspace symbols) are handled according to these rules in `Controller` classes.
```

Comment:
in [Controller](https://quarkiverse.github.io/quarkiverse-docs/quarkus-renarde/dev/index.html#_controllers) classes.